analysis/topEFT/make_cr_and_sr_plots.py

```diff
@@ -518,7 +522,8 @@ def main():
     if args.include_timestamp_tag:
         outdir_name = outdir_name + "_" + timestamp_tag
     save_dir_path = os.path.join(save_dir_path,outdir_name)
-    os.mkdir(save_dir_path)
+    if not os.path.exists:
+        os.mkdir(save_dir_path)
```

Comment:
Is this modification to prevent the code from complaining if the path already exists? Please correct me if I'm mixed up on this, but I think if the path *does* already exist, this change would mean that the old plots will be overwritten by the new ones. In that case, I actually prefer to code to raise an error if the path already exists. This prevents unintentionally overwriting other plots. For this reason, I'd prefer to leave the code as is. If you want to add the functionality to overwrite, I think we should add it as a command line option (and not have it turned on as the default behavior).

Reply:
Yes exactly. I introduced this when I was running the code a multitude of times during introducing the new SR for testing purposes. I'll remove it.